wannierberri/__system_tbmodels.py

```diff
@@ -42,11 +42,15 @@ def __init__(self,tbmodel=None,getAA=False,
                           frozen_max=-np.Inf,
                           random_gauge=False,
                           degen_thresh=-1 ,
+                          ksep=50,
```

Comment:
Could you add lines to the docstrings of all System* classes, to describe the new parameters ksep and delta_fz